### PR TITLE
Added Gropoint Air

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/dragino/groPointAirV1/lifecycle.schema.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "minimum": 0,
+      "maximum": 24,
+      "title": "Battery voltage",
+      "desciption": "The battery voltage in Volt.",
+      "unit": "V"
+    },
+    "sensorType": {
+      "title": "Sensor type",
+      "type": "integer",
+      "description": "Sensor type"
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1/lifecycle.schema.json:18]: add a description, what the different integers mean

File: types/dragino/groPointAirV1/meta.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "GroPoint Air",
+  "version": "1.0.0",
+  "manufacturer": "Dragino",
+  "url": "https://www.dragino.com/products/agriculture-weather-station/item/227-gropoint-air.html",
+  "description": "GroPoint Profile is plug-and-play with GroPoint Air to get soil moisture and send via wireless. GroPoint Profile is powered by GroPoint Air, no need external power source.",
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1/meta.json:6]: check this description for grammar

File: types/dragino/groPointAirV1/soil_moisture.schema.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "https://akenza.io/dragino/groPointAir/soil_moisture.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Soil moisture",
+  "type": "object",
+  "properties": {
+    "soilMoisture1": {
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1/soil_moisture.schema.json:7]: what does 1-8 mean?

File: types/dragino/groPointAirV1/uplink.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+  if (bytes.length - 3 === (temLen[senType] + humLen[senType]) * 2) {
+    isTemFlag = true;
+  }
+
+  let startBytes = 3;
+
+  if (isTemFlag === true) {
+    for (let i = 0; i < temLen[senType]; i++) {
+      if (bytes[startBytes] & 0x80) {
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1/uplink.js:57]: explain this if

File: types/dragino/groPointAirV1/uplink.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    "soilMoisture2",
+    "soilMoisture3",
+    "soilMoisture4",
+    "soilMoisture5",
+    "soilMoisture6",
+    "soilMoisture7",
+    "soilMoisture8",
+  ];
+
+  const temLen = [0, 0, 4, 6, 7, 9, 11, 12, 13];
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1/uplink.js:35]: use more descriptive variable names throughout this file

File: types/dragino/groPointAirV1_1/meta.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "GroPoint Air V1.1",
+  "version": "1.0.0",
+  "manufacturer": "Dragino",
+  "url": "https://www.dragino.com/products/agriculture-weather-station/item/227-gropoint-air.html",
+  "description": "GroPoint Profile is plug-and-play with GroPoint Air to get soil moisture and send via wireless. GroPoint Profile is powered by GroPoint Air, no need external power source.",
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1_1/meta.json:6]: check grammar

File: types/dragino/groPointAirV1_1/uplink.js [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    "soilMoisture2",
+    "soilMoisture3",
+    "soilMoisture4",
+    "soilMoisture5",
+    "soilMoisture6",
+    "soilMoisture7",
+    "soilMoisture8",
+  ];
+
+  const temLen = [0, 0, 4, 6, 7, 9, 11, 12, 13];
```
Comment (mariusgiger) [inline types/dragino/groPointAirV1_1/uplink.js:35]: same comments as above

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, please review the readability of this device type

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, please review the readability of this device type

File: types/dragino/groPointAirV1/uplink.js (hunk 5 above)
Comment (J2thatsme) [reply types/dragino/groPointAirV1/uplink.js:35]: Its a decoder without explanation from the sensor manufacturer.

File: types/dragino/groPointAirV1_1/uplink.js (hunk 7 above)
Comment (J2thatsme) [reply types/dragino/groPointAirV1_1/uplink.js:35]: same as above

**Review by mariusgiger — APPROVED** [review]
nothing to add ;) please link the source decoders

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**